frontend/src/data/tasks.json

```diff
@@ -590,7 +590,7 @@
       {
         "answerKey": "all",
         "descriptionEn": "After you apply, Service Canada will contact you by mail once a decision has been made, or to request additional information or documents. If you are approved, the letter will confirm:\n- your OAS payment start date (called the “effective date”)\n- the portion of the full OAS pension you are entitled to ([based on how long you have lived in Canada since age 18](https://www.canada.ca/en/services/benefits/publicpensions/cpp/old-age-security/benefit-amount.html#:~:text=If%20you%20have%20lived%20in%20Canada%20less%20than%2040%20years))\n- your monthly OAS payments at your start date\n- your first month's OAS payment amount (this may be different from your future monthly payments if you are receiving a retroactive payment)\n\nIf you qualify for GIS, the letter will also confirm:\n- your GIS payment start date (called the “effective date”)\n- your monthly GIS payments at your start date\n- your first month's GIS payment amount (this may be different from your future monthly payments if you are receiving a retroactive payment)",
```

Comment:
The URL here should probably be `https://www.canada.ca/en/services/benefits/publicpensions/cpp/old-age-security/benefit-amount.html`

The current URL uses a feature of Google Chrome that won't work in other browsers (`#:~:text=`).

Also, the french variant of this URL does not have the `#:~:text=` bit.

Reply:
Updated that link but there is a link underneath it that also uses that feature: https://www.canada.ca/en/employment-social-development/programs/old-age-security/reports/oas-toolkit.html#:~:text=When%20you%20can%20expect%20to%20hear%20from%20Service%20Canada"

Im reluctant to take it out entirely because it navigates to a section of the page that is near the bottom, and the page is very long. Is there a different URL bit we could use for this - it is line 598